index.html

```diff
@@ -80,10 +80,55 @@ <h1 class="intro_title">Discover Amazing places in Japan</h1>
       <section class="benefits">
         <div class="benefits_wrap">
           <h2 class="benefits_title">Benefits of Odigo</h2>
-          <div class="benefits_cards">
-            <div class="benefits_card"></div>
-            <div class="benefits_card"></div>
-            <div class="benefits_card"></div>
+          <div class="cards_container">
+            <div class="benefits_card">
+              <div class="benefits_card-pic">
+                <img
```

Comment:
Is it possible to get rid of this `div.benefits_card-pic wrapper` and apply its styles to image directly? I guess it will not affect the look of the layout  (I'm not sure, there is a need to check it), but the html can become more clean

The idea is: do not complicate things unnecessarily